src/EnergyPlus/DataOutputs.cc

```diff
@@ -167,7 +167,7 @@ bool FindItemInVariableList(EnergyPlusData &state, std::string const &KeyedValue
     // This function looks up a key and variable name value and determines if they are
     // in the list of required variables for a simulation.
 
-    auto const found_variable = state.dataOutput->OutputVariablesForSimulation.find(VariableName);
+    auto const found_variable = state.dataOutput->OutputVariablesForSimulation.find(std::string{VariableName});
```

Comment:
@Myoldmopar, the reason we have to construct a string here is that `OutputVariablesForSimulation` is a `std::unordred_map<std::string, ...>` which means that the argument passed to the `find` function is defined as `const std::string &`. (`std::string` is the argument to the template variable).

Apparently, there is a way to resolve this by declaring the `std::unordered_map` to use a transparent comparator (see [here](https://stackoverflow.com/questions/35525777/use-of-string-view-for-map-lookup) and [here](https://stackoverflow.com/questions/35525777/use-of-string-view-for-map-lookup)) but my template chops are not sufficient to figure out how to apply this solution.